### ISSUE TITLE
Feature request: Allow the use of MenuDivider inside MenuRadioGroup

### ISSUE THREAD

**Issue by jesuscc1993** [szhsin/react-menu#165]
Assuming I have a component rendering the following:
```js
<MenuRadioGroup
  value={appSettings.sortMode}
  onChange={onSortModeChange}
>
  <MenuItem value={ExplorerSortMode.Name}>Name</MenuItem>
  <MenuItem value={ExplorerSortMode.Kind}>Kind</MenuItem>
  <MenuDivider />
  <MenuItem value={ExplorerSortMode.Accessed}>Date accessed</MenuItem>
  <MenuItem value={ExplorerSortMode.Changed}>Date changed</MenuItem>
  <MenuItem value={ExplorerSortMode.Created}>Date created</MenuItem>
  <MenuItem value={ExplorerSortMode.Modified}>Date modified</MenuItem>
</MenuRadioGroup>
```

The `MenuDivider` will be ignored, with the following warning being logged in the terminal: `The permitted children inside a MenuRadioGroup are MenuItem`.

I see no reason why `MenuDivider` wouldn't be allowed here.

**Comment by szhsin** [comment]
Hi @jesuscc1993 , looks like this is a valid feature in some use cases and it will be added in a coming release.

Currently you might get around it by using the following structure:
```jsx
<Menu>
  <MenuRadioGroup
    value={appSettings.sortMode}
    onChange={onSortModeChange}>
    <MenuItem value={ExplorerSortMode.Name}>Name</MenuItem>
    <MenuItem value={ExplorerSortMode.Kind}>Kind</MenuItem>
  </MenuRadioGroup>
  <MenuDivider />
  <MenuRadioGroup
    value={appSettings.sortMode}
    onChange={onSortModeChange}>
    <MenuItem value={ExplorerSortMode.Accessed}>Date accessed</MenuItem>
    <MenuItem value={ExplorerSortMode.Changed}>Date changed</MenuItem>
    <MenuItem value={ExplorerSortMode.Created}>Date created</MenuItem>
    <MenuItem value={ExplorerSortMode.Modified}>Date modified</MenuItem>
  </MenuRadioGroup>
</Menu>
```
But be aware it could produce an incorrect [accessibility tree](https://www.w3.org/TR/wai-aria-1.1/#menuitemradio).

Only one of the items in a radio group can be checked at a time, so I think it's uncommon to have a separator dividing the group.  For example, on _macOS Big Sur_ these is a similar menu to your use case but it doesn't use a separator.

<img width="401" alt="sort" src="https://user-images.githubusercontent.com/41896553/122055953-7c0c5d00-ce2c-11eb-824b-ccde00e08d39.png">

**Comment by szhsin** [comment]
Issue fixed in v2.0.1
